src/components/safe-messages/MsgModal/CloseDialog.tsx

```diff
@@ -0,0 +1,31 @@
+import { type SetStateAction } from 'react'
+import { Button, Dialog, DialogActions, DialogContent, DialogContentText, DialogTitle } from '@mui/material'
+
+const CloseDialog = ({
```

Comment:
Can we avoid a popup on top of another popup?
I would show a tooltip next to the submit/close button instead.
<img width="333" alt="Screenshot 2023-04-18 at 09 45 56" src="https://user-images.githubusercontent.com/381895/232693848-4d9640ce-a2e2-481d-9d9f-d4952f2865c9.png">

Reply:
What is the case we want to warn in the "Submit" button? this modal only has "Sign"

Reply:
Let's leave the UX details out of scope for now.
The main purpose of this PR should be to make the flow functional for m/n Safe Accounts.
 
I will create an issue about necessary design / UX changes needed for the synch flow.